atomic_reactor/core.py

```diff
@@ -158,9 +159,12 @@ def build_image_dockerhost(self, build_image, json_args_path):
                 open(os.path.join(json_args_path, BUILD_JSON)).read())
         container_id = self.tasker.run(
             ImageName.parse(build_image),
-            create_kwargs={'volumes': [DOCKER_SOCKET_PATH, json_args_path]},
-            start_kwargs={'binds': volume_bindings,
-                          'privileged': True}
+            create_kwargs={'volumes': [DOCKER_SOCKET_PATH, json_args_path],
+                           'host_config': create_host_config (
+                               binds = volume_bindings,
+                               privileged =True
```

Comment:
Minor nit: I don't think we want spaces before '(' or around '=' in these three lines, and likewise for the other change.